### ISSUE TITLE
constant Net::HTTPServerException is deprecated

### ISSUE THREAD

**Issue by oliverklee** [w3c-validators/w3c_validators#43]
Currently, the W3C validation service seems to be down (error 503 when trying to validate a page using the web form or the gem). Hence, the gem cannot validate any page (which is okay).

However, this also uncovered use of a deprecated constant:

```
…/lib/w3c_validators/validator.rb:177: warning: constant Net::HTTPServerException is deprecated
```

**Comment by doc75** [comment]
@oliverklee thanks for reporting this warning. Will be fixed in version 1.3.5 currently under delivery.

**Comment by doc75** [comment]
Indeed, this cannot be fixed for now since it works on Ruby 2.6 but not on previous version still supported (2.5 and 2.4).
I'll keep it open and probably fix it when 2.5 end of life happened.
